client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1259,6 +1260,7 @@
   "label.units-available": "{{unit}} Available",
   "label.units-in-stock": "{{unit}} in Stock",
   "label.units-issued": "{{unit}} Issued",
+  "label.issued": "Issued",
```

Comment:
These got changed recently, so the strings in the Prescription component were broken.